src/commands/deploy.ts

```diff
@@ -4,6 +4,7 @@ import { Message, Client } from 'discord.js';
 
 export default {
     name: 'deploy',
+    level: 'owner',
```

Comment:
Further adding on from my 2 previous comments, I think that in this case, the `ownerOnly` boolean can be used instead of `level: owner`.